src/testUtils/java/uk/gov/hmcts/reform/wataskmanagementapi/config/GivensBuilder.java

```diff
@@ -151,7 +152,7 @@ public GivensBuilder iCreateATaskWithCaseId(String caseId, String taskType) {
                     result.then().assertThat()
                         .statusCode(HttpStatus.OK.value())
                         .contentType(APPLICATION_JSON_VALUE)
-                        .body("size()", is(1));
+                        .body("size()", greaterThanOrEqualTo(1));
```

Comment:
Can't really tell what this method is used for but seems like it could potentially affect tests that expect exactly 1 response (if there are any)?  Is there any reason for changing the expectation here?

Reply:
Reverted.